napalm_eos/eos.py

```diff
@@ -795,9 +795,9 @@ def get_arp_table(self):
 
         return arp_table
 
-    def get_ntp_peers(self):
+    def get_ntp_stats(self):
 
-        ntp_peers = dict()
+        ntp_stats = list()
```

Comment:
But if needed, it takes just a couple of seconds.

Thanks for sharing, very interesting! :)